### PR TITLE
API Explorer tweaks

### PR DESCRIPTION
**Pull request by spadgett**
* Move under Home nav section to make it more discoverable
* Add Namspaced and Description columns to table

cc @smarterclayton 
/assign @rhamilto 

@bmignano Let me know if you object to these changes

Screen Shot 2019-08-13 at 3 03 52 PM

### CODE REVIEW

File: frontend/public/style/mixin/_line-clamp.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@mixin co-line-clamp($lines) {
```
Comment (rhamilto) [inline frontend/public/style/mixin/_line-clamp.scss]: Add a default value so `$lines` is optional
```suggestion
@mixin co-line-clamp($lines: 3) {
```

**Comment by spadgett** [comment]
@rhamilto updated

**Comment by bmignano** [comment]
Makes sense to me! Curious if that text box filter needs to be `Filter by kind or group...` still since we have that Group filter dropdown now. Seems like we could just leave it at `Filter by kind...`?

**Comment by spadgett** [comment]
@rhamilto fixed the linter error, another look?

**Comment by smarterclayton** [comment]
I still am partial to "Explore" over "API Explorer".  If necessary we could
use a slightly longer H1 on the page "Explore All Resources"

**Comment by spadgett** [comment]
Updated the nav item to "Explore" and changed the text filter to only filter kind.

Screen Shot 2019-08-13 at 6 58 21 PM

**Comment by spadgett** [comment]
Added a fix to remember selected filters using the back button

**Pull request merged**